Add a default folder

MagTun opened the pull request:
(I apply the first change "Handle leading or trailing spaces" to my the "main" branch on my fork so it is also add to this pull request. Sorry for that!)

MagTun committed:
Handle leading or trailing spaces

MagTun committed:
add a default folder

MagTun committed:
add a default folder

MagTun committed:
add a default folder

MagTun committed:
better way for default folder

This alternative may be better as it calls the file explorer pop up set to the default path.

Asu4ni commented:
Hi @MagTun, I appreciate your work to improve the addon.

Before I merge it, could you please make your commits in this PR into one commit (`git rebase -i`), and hopefully with a clear commit message illustrating what you've added? Also for #3.

Again, thanks for the contribution. I'll merge #1 soon and update it on Anki web once all three of your PRs are merged.

Asu4ni commented on templates.py:
```diff
@@ -25,12 +25,15 @@ def _reload_config():
     _tmpl_ext = utils.cfg("tmplExt")
     _merge_css = utils.cfg("mergeCSS")
-
+    _defaultfolder = utils.cfg("defaultfolder")
```
please also make `_defaultfolder` global: add it to line 23

MagTun committed:
added _defaultfolder  to global line 23

MagTun commented:
Sorry for this mistake with the global var. 

I am not using git (I have edited the file manually on github) and I can't find a way to use something similar to `git rebase -i` directly on github. 

I saw that you were able to merge the main branch. Should I delete this branch and commit to the main one? (Sorry I don't know a lot about pull requests). I also see this pull request has a conflict but I don't understand why. 

For this pull request:
I have added a variable "_defaultfolder " that is used for the folder explorer that is opened for export/import (before it was the anki collection.media which is ideal. 

For pull request 3:
I have added a subfolder on the export with the current date time to prevent overwriting previous exported templates.
